tests/integration/ceph_smoke_test.go

```diff
@@ -249,7 +249,7 @@ func (suite *SmokeSuite) TestPoolResize() {
 	require.Equal(suite.T(), true, poolResized, fmt.Sprintf("pool %s not found", poolName))
 
 	// Verify the Kubernetes Secret has been created (bootstrap peer token)
-	pool, err := suite.k8sh.RookClientset.CephV1().CephBlockPools(suite.namespace).Get(poolName, metav1.GetOptions{})
+	pool, _ := suite.k8sh.RookClientset.CephV1().CephBlockPools(suite.namespace).Get(poolName, metav1.GetOptions{})
```

Comment:
ditto do not silence the error, handle it

Reply:
Done.